### ISSUE TITLE
Run the script on Netscaler

### ISSUE THREAD

**Issue by crisb811** [certat/citrix-logchecker#1]
Does anyone know how to run this script on the Netscaler?

**Comment by ashwin2101** [comment]
I don't think so but i'am also missing information about how to run this script?
Do we have to download al the netscaler log files to a linux machine?
